src/bijectors/stacked.jl

```diff
@@ -133,7 +117,9 @@ end
 #     logjac += sum(_logjac)
 #     return (rv = vcat(y_1, y_2), logabsdetjac = logjac)
 # end
-@generated function forward(b::Stacked{T, N}, x::AbstractVector) where {N, T<:Tuple}
+@generated function forward(b::Stacked{T}, x::AbstractVector) where {T<:Tuple}
+    N = length(T.parameters)
+    
```

Comment:
An alternative I assume:
```suggestion
@generated function forward(b::Stacked{<:Tuple{Vararg{<:Any,N}}}, x::AbstractVector) where {N}
```

Reply:
Yeah, that's maybe nicer :+1: